sorc/prepobs_prepdata.fd/prepdata.f

```diff
@@ -12672,7 +12672,8 @@ SUBROUTINE SATEDS
       CHARACTER*8   STNPRT,STNID,DSNAME,SUBSET_d
       LOGICAL  SATMST,TR80KM,TOVEDS
       INTEGER  IBUF(140),JRTRV(3),NUMOB(5,4,2,4,4),KEPOB(5,4,2,4),
-     $ MSTOB(5,4,2,4),IDATA(MAXOBS),KSATOB(3),IDSAT(4),ISAMPLE(2)
+     $ MSTOB(5,4,2,4),IDATA(MAXOBS),KSATOB(3),IDSAT(8),ISAMPLE(2) 
+C     MADE IDSAT(4) TO IDSAT(8) to accomodate compiler on WCOSS2     
```

Comment:
Put this comment up above around line 12631 in the official documentation timeline. Follow the formatting already in place.

Reply:
Also add comment after line 1337 in main documentation block to indicate this change to the SATEDS routine. Follow the formatting previously established.

Reply:
Also update the W3TAGB parameters in line 3109 to be 2021,0105,0050,'EMC'

Reply:
My bad ... parameters should be 2022, not 2021

Reply:
Done! I see we use JDay in W3TAGB :)

Reply:
Its day of year ... not sure if that is JDay.